bin/zopen-importenvs

```diff
@@ -1,18 +1,87 @@
-#!/bin/sh
-#
+#!/bin/bash
```

Comment:
We may want to change this to /bin/env bash in case bash is not found in /bin since I don't believe it comes with the OS by default.

Also, add a comment that bash is required because of BASH_SOURCE - to locate the path of the sourced shell script

Reply:
+1 - we should always use 'sh' not 'bash'